config/checker_severity_map.json

```diff
@@ -118,10 +121,12 @@
   "cert-msc32-c":                                               "MEDIUM",
   "cert-msc50-cpp":                                             "LOW",
   "cert-msc51-cpp":                                             "MEDIUM",
+  "cert-mem57-cpp":                                             "MEDIUM",
   "cert-oop11-cpp":                                             "MEDIUM",
   "cert-oop54-cpp":                                             "MEDIUM",
   "cert-oop57-cpp":                                             "HIGH",
-  "cert-oop58-cpp":                                             "LOW",
+  "cert-oop58-cpp":                                             "MEDIUM",
```

Comment:
The severity level of this checker at [here](https://wiki.sei.cmu.edu/confluence/display/cplusplus/OOP58-CPP.+Copy+operations+must+not+mutate+the+source+object) is `LOW`.

Reply:
I followed our own guidlines:
The following severity levels are defined:

- **STYLE**: A true positive indicates that the source code is against a specific coding guideline or could improve readability.
Example: LLVM Coding Guideline: Do not use else or else if after something that interrupts control flow (break, return, throw, continue).

- **LOW**: A true positive indicates that the source code is hard to read/understand or could be easily optimized.
Example: Unused variables, Dead code.

- **MEDIUM**: A true positive indicates that the source code that may not cause a run-time error (yet), but against intuition and hence prone to error.
Example: Redundant expression in a condition.

- **HIGH**: A true positive indicates that the source code will cause a run-time error.
  Example of this category: out of bounds array access, division by zero, memory leak.

- **CRITICAL**: Currently unused. This severity level is reserved for later use.

Reply:
according to this, I think cert-oop58-cpp and cert-pos44-c should be medium, because if it is a true positive it will cause a runtime error.